javascript-functions-practice/get-first-character.js

```diff
@@ -0,0 +1,5 @@
+/* exported getFirstCharacter */
+function getFirstCharacter(string) {
+  var firstCharacter = string.charAt(0);
```

Comment:
worth pointing out you have also do 
string[0]

Reply:
haha, i just did what i remembered from before... noted :)